content/blog/Stochastic-process-1.md

```diff
@@ -0,0 +1,80 @@
+---
+title       : "Stochastic Process Part 1 - Introduction to Stochastic Processes"
+date        : 2020-09-17T10:00:01+05:30
+description : "Getting an understanding of Stochastic processes with a basic example"
+feature_image: "/images/blog/mathematics/blog_2/main.jpg"
+image: "/images/blog/mathematics/blog_2/main.jpg"
+author: Inderjit Singh Chahal
+tags: Mathematics Probability
+---
+
+In this part 1, we will look at getting a better understanding of a stochastic process, in the next one we will look at some practical implementations to build on this understanding to solve some real-world problems. 
+
+
+### What is a Stochastic Process
+
+Wikipedia defines a stochastic process as a mathematical object that is defined by a family of random variables.
+
+A better way to look at it would be
+
+**It is a collection of random “events”, each happening with either fixed or varying probabilities, over a given time**
```

Comment:
Nothing wrong with this, but just fyi, you can use quotes to quote things. Markdown syntax for that is lines starting with ">" e.g

```
> I am a quote
```
becomes
> I am a quote